digitalpublicgoods/global-open-facility-registry.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "Global Open Facility Registry",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/intrahealth/gofr"
```

Comment:
This link does not provide evidence for clear ownership. Please provide a more specific link

Reply:
Here’s the primary website on GOFR that is owned and operated by IntraHealth and its publicly available information which therefore fits the clear ownership indicator of the DPG standard.

Resolved in commit 43230d3